src/components/BottomBar/KeybindingPanel.jsx

```diff
@@ -154,39 +145,36 @@ class KeybindingPanel extends Component {
   }
 
   checkForModifiers = (action) => {
-    var showForModifiers = false;
-    var modifierObject = {
+    let modifierObject = {
       alt: this.state.activeModifiers.includes('alt'),
       control: this.state.activeModifiers.includes('control'),
-      shift: this.state.activeModifiers.includes('shift')
-    }
-    modifierObject["super"] = this.state.activeModifiers.includes('super');
-    var hadModifier = (action.modifiers["super"] == false) && (action.modifiers["alt"] == false) && (action.modifiers["control"] == false) && (action.modifiers["shift"] == false);
-    hadModifier = !hadModifier;
-    showForModifiers = Object.entries(modifierObject).toString() === Object.entries(action.modifiers).toString();
-    if (showForModifiers || (!hadModifier && this.state.activeModifiers.length == 0)) {
-      return true;
+      shift: this.state.activeModifiers.includes('shift'),
+      super: this.state.activeModifiers.includes('super')
     }
-    return false;
+    const hasModifier = (action.modifiers["super"] || action.modifiers["alt"] || 
+                         action.modifiers["control"] || action.modifiers["shift"]);
+
+    const showForModifiers = Object.entries(modifierObject).toString() === Object.entries(action.modifiers).toString();
+
+    return showForModifiers || (!hasModifier && this.state.activeModifiers.length == 0);
```

Comment:
Make the second part of the statement into bool? A little hard to understand what it means currently